app/Startup/Models/Account.php

```diff
@@ -2,21 +2,37 @@
 
 namespace Startup\Models;
 
+use Illuminate\Database\Eloquent\Concerns\HasTimestamps;
 use Illuminate\Database\Eloquent\Model;
 use Laravel\Cashier\Billable;
 use Startup\Traits\HasSubscriptions;
 use Startup\Traits\HasTeam;
 
-class Account extends Model
+/**
+ * @property    int                             $id
+ * @property    string                          name
+ * @property    int                             $owner_id
+ * @property    string|null                     $billing_name
+ * @property    string|null                     $stripe_id
+ * @property    string|null                     $card_brand
+ * @property    string|null                     $card_last_four
+ * @property    \Carbon\Carbon                  $trial_ends_at
+ * @property    \Carbon\Carbon                  $created_at
+ * @property    \Carbon\Carbon                  $updated_at
+ */
+class Account extends Model implements \JsonSerializable
 {
-    use Billable, HasSubscriptions, HasTeam;
+    use Billable, HasSubscriptions, HasTeam, HasTimestamps;
```

Comment:
Doesn't the base Eloquent class already use this?

Reply:
Hah, I can't believe I never saw that. You're right. I'll make the changes and get rid of it.